site/guide/organize-model-inventory.qmd

```diff
@@ -0,0 +1,28 @@
+---
+title: "Organize models in the inventory"
+---
+
+Customize the model inventory to fit your needs.
```

Comment:
Short 'n' sweet and to the point, I see. 😁  Can you maybe expand this a little bit to put some context around "your needs". E.g. one thing that I can see that is immediately helpful is that I can filter out models so that I see only the models that are owned by my business unit.

(Can you also check if the filters work correctly for you, I was not able to filter on some of the available business units which feels bug-ish to me.)

Reply:
No bugs on my end for the filter feature! I edited the description but I still feel like it's too blunt